.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
I'm not a huge fan of persistent .files. Is this necessary or could we just make it a real name?